ggml-sycl.cpp

```diff
@@ -9455,24 +10776,385 @@ static void convert_mul_mat_vec_f16_sycl(const void *vx, const dfloat *y,
     }
 }
 
-template <int qk, int qi, typename block_q_t, int vdr,
-          vec_dot_q_sycl_t vec_dot_q_sycl>
-static void mul_mat_vec_q_sycl_submitter(const void *vx, const void *vy,
+
+static void mul_mat_vec_q4_0_q8_1_sycl(const void *vx, const void *vy,
```

Comment:
Can we rewrite the function name as ```mul_mat_vec_q4_0_sycl``` to make it consistent with cuda

Reply:
This function name is referred to original CUDA code when migration.
I will rename it after the SYCL backend